com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/graph.py

```diff
@@ -18,7 +18,10 @@ def __init__(self, name=None):
     def addOperator(self, kind, function=None, name=None):
         if name is None:
             name = self.name + "_OP"+str(len(self.operators))
-        op = SPLInvocation(len(self.operators), kind, function, name, {}, self)
+        if (kind == "$Isolate"):
```

Comment:
that makes sense. I will make that change once I determine my GitHub usage issues